clang/test/3C/hash.c

```diff
@@ -9,8 +9,14 @@
 #include <stdlib.h>
 #include <string.h>
 
-_Itype_for_any(T) void vsf_sysutil_memclr(void *p_dest : itype(_Array_ptr<T>) byte_count(size), unsigned int size)
-// CHECK_ALL: _Itype_for_any(T) void vsf_sysutil_memclr(void *p_dest : itype(_Array_ptr<T>) byte_count(size), unsigned int size)
+_Itype_for_any(T) void vsf_sysutil_memclr(void *p_dest
+                                          : itype(_Array_ptr<T>)
+                                                byte_count(size),
+                                            unsigned int size)
+// CHECK_ALL: _Itype_for_any(T) void vsf_sysutil_memclr(void *p_dest
+// CHECK-NEXT: : itype(_Array_ptr<T>)
+// CHECK-NEXT: byte_count(size),
+// CHECK-NEXT: unsigned int size)
```

Comment:
Ah, I missed that you're using a custom check prefix, so I think this should be:
```suggestion
// CHECK_ALL-NEXT: : itype(_Array_ptr<T>)
// CHECK_ALL-NEXT: byte_count(size),
// CHECK_ALL-NEXT: unsigned int size)
```
Example:
https://github.com/correctcomputation/checkedc-clang/blob/cf35b121de3211cbcac3e107edacaaf8c89e45e0/clang/test/3C/stmtExprA.c#L16-L20